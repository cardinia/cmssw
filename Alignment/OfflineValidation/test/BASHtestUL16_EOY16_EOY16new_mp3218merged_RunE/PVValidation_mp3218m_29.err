26-Sep-2019 12:57:58 CEST  Initiating request to open file root://eoscms.cern.ch//eos/cms/store/data/Run2016E/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/10000/2E1A6712-E207-E911-87C9-008CFAC93E04.root
26-Sep-2019 12:58:02 CEST  Successfully opened file root://eoscms.cern.ch//eos/cms/store/data/Run2016E/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/10000/2E1A6712-E207-E911-87C9-008CFAC93E04.root
Begin processing the 1st record. Run 277127, Event 1354984232, LumiSection 720 on stream 0 at 26-Sep-2019 12:58:13.053 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 26-Sep-2019 13:10:49 CEST  Run: 277127 Event: 1362068854
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 0
%MSG
Begin processing the 1001st record. Run 277127, Event 1362322444, LumiSection 724 on stream 0 at 26-Sep-2019 13:11:06.542 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 26-Sep-2019 13:13:31 CEST  Run: 277127 Event: 1366492500
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 12020
%MSG
Begin processing the 2001st record. Run 277127, Event 1369857600, LumiSection 728 on stream 0 at 26-Sep-2019 13:16:16.781 CEST
Begin processing the 3001st record. Run 277127, Event 1378230850, LumiSection 733 on stream 0 at 26-Sep-2019 13:21:19.678 CEST
Begin processing the 4001st record. Run 277127, Event 1387098697, LumiSection 738 on stream 0 at 26-Sep-2019 13:26:27.967 CEST
Begin processing the 5001st record. Run 277127, Event 1394425626, LumiSection 742 on stream 0 at 26-Sep-2019 13:31:34.680 CEST
Begin processing the 6001st record. Run 277127, Event 1403720529, LumiSection 747 on stream 0 at 26-Sep-2019 13:36:50.336 CEST
Begin processing the 7001st record. Run 277127, Event 1411622539, LumiSection 752 on stream 0 at 26-Sep-2019 13:42:00.715 CEST
Begin processing the 8001st record. Run 277127, Event 1418581178, LumiSection 756 on stream 0 at 26-Sep-2019 13:47:15.829 CEST
Begin processing the 9001st record. Run 277127, Event 1428086961, LumiSection 761 on stream 0 at 26-Sep-2019 13:52:23.055 CEST
Begin processing the 10001st record. Run 277127, Event 1434495801, LumiSection 765 on stream 0 at 26-Sep-2019 13:57:25.136 CEST
Begin processing the 11001st record. Run 277127, Event 1442306455, LumiSection 770 on stream 0 at 26-Sep-2019 14:02:28.170 CEST
Begin processing the 12001st record. Run 277127, Event 1451990910, LumiSection 775 on stream 0 at 26-Sep-2019 14:07:28.630 CEST
Begin processing the 13001st record. Run 277127, Event 1459029516, LumiSection 779 on stream 0 at 26-Sep-2019 14:11:53.077 CEST
Begin processing the 14001st record. Run 277127, Event 1467827573, LumiSection 784 on stream 0 at 26-Sep-2019 14:14:54.895 CEST
Begin processing the 15001st record. Run 277127, Event 1476469381, LumiSection 789 on stream 0 at 26-Sep-2019 14:17:56.307 CEST
Begin processing the 16001st record. Run 277127, Event 1483430594, LumiSection 793 on stream 0 at 26-Sep-2019 14:20:54.111 CEST
Begin processing the 17001st record. Run 277127, Event 1492328268, LumiSection 798 on stream 0 at 26-Sep-2019 14:23:50.497 CEST
Begin processing the 18001st record. Run 277127, Event 1498055398, LumiSection 802 on stream 0 at 26-Sep-2019 14:26:48.208 CEST
Begin processing the 19001st record. Run 277127, Event 1506228028, LumiSection 807 on stream 0 at 26-Sep-2019 14:29:45.219 CEST
Begin processing the 20001st record. Run 277127, Event 1516443847, LumiSection 812 on stream 0 at 26-Sep-2019 14:32:48.416 CEST
Begin processing the 21001st record. Run 277127, Event 1523960225, LumiSection 817 on stream 0 at 26-Sep-2019 14:35:56.413 CEST
Begin processing the 22001st record. Run 277127, Event 1533002521, LumiSection 822 on stream 0 at 26-Sep-2019 14:38:54.455 CEST
Begin processing the 23001st record. Run 277127, Event 1541450936, LumiSection 827 on stream 0 at 26-Sep-2019 14:41:56.167 CEST
Begin processing the 24001st record. Run 277127, Event 1547192124, LumiSection 831 on stream 0 at 26-Sep-2019 14:44:55.682 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 26-Sep-2019 14:47:07 CEST  Run: 277127 Event: 1555418312
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 0
%MSG
Begin processing the 25001st record. Run 277127, Event 1557124122, LumiSection 836 on stream 0 at 26-Sep-2019 14:47:57.925 CEST
Begin processing the 26001st record. Run 277127, Event 1565096200, LumiSection 841 on stream 0 at 26-Sep-2019 14:50:57.627 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 26-Sep-2019 14:52:55 CEST  Run: 277127 Event: 1570681358
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 7552
%MSG
Begin processing the 27001st record. Run 277127, Event 1573924976, LumiSection 846 on stream 0 at 26-Sep-2019 14:53:56.121 CEST
Begin processing the 28001st record. Run 277127, Event 1579336440, LumiSection 850 on stream 0 at 26-Sep-2019 14:56:57.879 CEST
Begin processing the 29001st record. Run 277127, Event 1588696183, LumiSection 855 on stream 0 at 26-Sep-2019 14:59:59.526 CEST
Begin processing the 30001st record. Run 277127, Event 1597604655, LumiSection 859 on stream 0 at 26-Sep-2019 15:03:00.731 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 26-Sep-2019 15:03:36 CEST  Run: 277127 Event: 1597394935
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 2380
%MSG
Begin processing the 31001st record. Run 277127, Event 1603504905, LumiSection 862 on stream 0 at 26-Sep-2019 15:06:03.808 CEST
Begin processing the 32001st record. Run 277127, Event 1611730290, LumiSection 866 on stream 0 at 26-Sep-2019 15:09:08.741 CEST
Begin processing the 33001st record. Run 277127, Event 1621195682, LumiSection 870 on stream 0 at 26-Sep-2019 15:12:09.725 CEST
26-Sep-2019 15:14:19 CEST  Closed file root://eoscms.cern.ch//eos/cms/store/data/Run2016E/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/10000/2E1A6712-E207-E911-87C9-008CFAC93E04.root
26-Sep-2019 15:14:19 CEST  Initiating request to open file root://eoscms.cern.ch//eos/cms/store/data/Run2016E/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/10000/8C1385E6-8407-E911-9504-002590D9D8C4.root
26-Sep-2019 15:14:22 CEST  Successfully opened file root://eoscms.cern.ch//eos/cms/store/data/Run2016E/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/10000/8C1385E6-8407-E911-9504-002590D9D8C4.root
Begin processing the 34001st record. Run 277127, Event 1627050064, LumiSection 873 on stream 0 at 26-Sep-2019 15:15:20.098 CEST
Begin processing the 35001st record. Run 277127, Event 1636120602, LumiSection 877 on stream 0 at 26-Sep-2019 15:18:17.729 CEST
Begin processing the 36001st record. Run 277127, Event 1643501242, LumiSection 881 on stream 0 at 26-Sep-2019 15:21:14.135 CEST
Begin processing the 37001st record. Run 277127, Event 1652233982, LumiSection 885 on stream 0 at 26-Sep-2019 15:25:33.454 CEST
Begin processing the 38001st record. Run 277127, Event 1659552754, LumiSection 889 on stream 0 at 26-Sep-2019 15:30:22.327 CEST
Begin processing the 39001st record. Run 277127, Event 1666027356, LumiSection 892 on stream 0 at 26-Sep-2019 15:34:59.431 CEST
Begin processing the 40001st record. Run 277127, Event 1675337344, LumiSection 896 on stream 0 at 26-Sep-2019 15:39:23.359 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 26-Sep-2019 15:39:57 CEST  Run: 277127 Event: 1676391572
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 0
%MSG
Begin processing the 41001st record. Run 277127, Event 1683516498, LumiSection 900 on stream 0 at 26-Sep-2019 15:44:30.268 CEST
26-Sep-2019 15:47:33 CEST  Closed file root://eoscms.cern.ch//eos/cms/store/data/Run2016E/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/10000/8C1385E6-8407-E911-9504-002590D9D8C4.root
26-Sep-2019 15:47:33 CEST  Initiating request to open file root://eoscms.cern.ch//eos/cms/store/data/Run2016E/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/10000/2C14F426-8507-E911-B66F-008CFAEBDC04.root
26-Sep-2019 15:47:37 CEST  Successfully opened file root://eoscms.cern.ch//eos/cms/store/data/Run2016E/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/10000/2C14F426-8507-E911-B66F-008CFAEBDC04.root
Begin processing the 42001st record. Run 277127, Event 531696744, LumiSection 284 on stream 0 at 26-Sep-2019 15:49:55.307 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 26-Sep-2019 15:50:26 CEST  Run: 277127 Event: 534753306
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 2478
%MSG
Begin processing the 43001st record. Run 277127, Event 552680782, LumiSection 296 on stream 0 at 26-Sep-2019 15:55:08.581 CEST
Begin processing the 44001st record. Run 277127, Event 560076248, LumiSection 300 on stream 0 at 26-Sep-2019 16:00:28.139 CEST
Begin processing the 45001st record. Run 277127, Event 569053290, LumiSection 305 on stream 0 at 26-Sep-2019 16:05:44.833 CEST
Begin processing the 46001st record. Run 277127, Event 578312788, LumiSection 310 on stream 0 at 26-Sep-2019 16:10:59.988 CEST
Begin processing the 47001st record. Run 277127, Event 586185725, LumiSection 315 on stream 0 at 26-Sep-2019 16:16:04.088 CEST
Begin processing the 48001st record. Run 277127, Event 592783265, LumiSection 319 on stream 0 at 26-Sep-2019 16:21:11.071 CEST
Begin processing the 49001st record. Run 277127, Event 601338775, LumiSection 324 on stream 0 at 26-Sep-2019 16:26:26.771 CEST
26-Sep-2019 16:31:48 CEST  Closed file root://eoscms.cern.ch//eos/cms/store/data/Run2016E/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/10000/2C14F426-8507-E911-B66F-008CFAEBDC04.root

=============================================

MessageLogger Summary

 type     category        sev    module        subroutine        count    total
 ---- -------------------- -- ---------------- ----------------  -----    -----
    1 OutOfBounds          -w TrackRefitter:Fi                       7        7
    2 fileAction           -s file_close                             3        3
    3 fileAction           -s file_open                              6        6

 type    category    Examples: run/evt        run/evt          run/evt
 ---- -------------------- ---------------- ---------------- ----------------
    1 OutOfBounds          277127/136206885 277127/136649250 277127/534753306
    2 fileAction           PostProcessEvent PostProcessEvent PostGlobalEndRun
    3 fileAction           pre-events       pre-events       PostProcessEvent

Severity    # Occurrences   Total Occurrences
--------    -------------   -----------------
Warning                 7                   7
System                  9                   9

dropped waiting message count 0
[9.851MB/9.851MB][100%][==================================================][9.851MB/s]  [9.851MB/9.851MB][100%][==================================================][9.851MB/s]  
