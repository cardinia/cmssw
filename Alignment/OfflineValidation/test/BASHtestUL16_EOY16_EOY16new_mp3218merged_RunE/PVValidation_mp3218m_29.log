000 (3189561.029.000) 09/26 11:23:32 Job submitted from host: <137.138.44.75:9618?addrs=137.138.44.75-9618+[2001-1458-d00-1a--b]-9618&alias=bigbird14.cern.ch&noUDP&sock=schedd_2873_37ae_40>
...
040 (3189561.029.000) 09/26 12:57:39 Started transferring input files
	Transferring to host: <188.185.172.20:9618?addrs=188.185.172.20-9618+[2001-1459-301-5a--100-b]-9618&noUDP&sock=17312_9eb9_73229>
...
040 (3189561.029.000) 09/26 12:57:39 Finished transferring input files
...
001 (3189561.029.000) 09/26 12:57:39 Job executing on host: <188.185.172.20:9618?addrs=188.185.172.20-9618+[2001-1459-301-5a--100-b]-9618&noUDP&sock=17178_41f6_3>
...
006 (3189561.029.000) 09/26 12:57:48 Image size of job updated: 125484
	69  -  MemoryUsage of job (MB)
	69740  -  ResidentSetSize of job (KB)
...
006 (3189561.029.000) 09/26 13:02:49 Image size of job updated: 552704
	430  -  MemoryUsage of job (MB)
	440196  -  ResidentSetSize of job (KB)
...
006 (3189561.029.000) 09/26 13:12:08 Image size of job updated: 666948
	539  -  MemoryUsage of job (MB)
	551304  -  ResidentSetSize of job (KB)
...
006 (3189561.029.000) 09/26 13:25:07 Image size of job updated: 667068
	539  -  MemoryUsage of job (MB)
	551424  -  ResidentSetSize of job (KB)
...
006 (3189561.029.000) 09/26 13:35:30 Image size of job updated: 668012
	540  -  MemoryUsage of job (MB)
	552360  -  ResidentSetSize of job (KB)
...
006 (3189561.029.000) 09/26 13:41:44 Image size of job updated: 668104
	540  -  MemoryUsage of job (MB)
	552452  -  ResidentSetSize of job (KB)
...
006 (3189561.029.000) 09/26 14:06:47 Image size of job updated: 669396
	541  -  MemoryUsage of job (MB)
	553744  -  ResidentSetSize of job (KB)
...
006 (3189561.029.000) 09/26 14:16:48 Image size of job updated: 669936
	542  -  MemoryUsage of job (MB)
	554284  -  ResidentSetSize of job (KB)
...
006 (3189561.029.000) 09/26 14:36:51 Image size of job updated: 671156
	543  -  MemoryUsage of job (MB)
	555504  -  ResidentSetSize of job (KB)
...
006 (3189561.029.000) 09/26 15:16:54 Image size of job updated: 675220
	556  -  MemoryUsage of job (MB)
	568360  -  ResidentSetSize of job (KB)
...
006 (3189561.029.000) 09/26 15:21:54 Image size of job updated: 675220
	556  -  MemoryUsage of job (MB)
	568380  -  ResidentSetSize of job (KB)
...
006 (3189561.029.000) 09/26 15:26:55 Image size of job updated: 675220
	556  -  MemoryUsage of job (MB)
	568412  -  ResidentSetSize of job (KB)
...
006 (3189561.029.000) 09/26 15:43:27 Image size of job updated: 675220
	557  -  MemoryUsage of job (MB)
	569556  -  ResidentSetSize of job (KB)
...
006 (3189561.029.000) 09/26 16:18:00 Image size of job updated: 675220
	569  -  MemoryUsage of job (MB)
	581716  -  ResidentSetSize of job (KB)
...
040 (3189561.029.000) 09/26 16:32:54 Started transferring output files
...
040 (3189561.029.000) 09/26 16:32:54 Finished transferring output files
...
005 (3189561.029.000) 09/26 16:32:55 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 03:17:08, Sys 0 00:00:13  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 03:17:08, Sys 0 00:00:13  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	11925  -  Run Bytes Sent By Job
	1015  -  Run Bytes Received By Job
	11925  -  Total Bytes Sent By Job
	1015  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :                 1           
	   Disk (KB)            :    10138        1   3765131 
	   Memory (MB)          :      569     2000      2000 
...
