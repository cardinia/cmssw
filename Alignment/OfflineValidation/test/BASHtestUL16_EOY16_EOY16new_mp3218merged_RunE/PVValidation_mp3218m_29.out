Job started at  Thu Sep 26 12:57:40 CEST 2019
batch dir: /pool/condor/dir_6365 release: /afs/cern.ch/cms/CAF/CMSALCA/ALCA_TRACKERALIGN/data/commonValidation/alignmentObjects/paconnor/CMSSW_10_6_0/src release base: /cvmfs/cms.cern.ch/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_0
cmsRun PVValidation_mp3218m_277127_cfg.py
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: This is DATA!
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: Applying TrackerSurfaceDeformations!
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: Running DA Algorithm!
Content of working dir is total 10M -rw-r--r--. 1 tkello zh 9.9M Sep 26 16:32 PVValidation_mp3218m_277127.root -rw-r--r--. 1 tkello zh 21K Sep 26 12:57 PVValidation_mp3218m_277127_cfg.py -rw-r--r--. 1 tkello zh 11K Sep 26 16:32 _condor_stderr -rw-r--r--. 1 tkello zh 521 Sep 26 12:57 _condor_stdout -rwxr-xr-x. 1 tkello zh 1015 Sep 26 12:57 condor_exec.exe -rw-------. 1 tkello zh 4.3K Sep 26 12:57 tkello.cc drwx------. 3 tkello zh 4.0K Sep 26 12:57 tmp drwx------. 3 tkello zh 4.0K Sep 26 12:57 var
Job ended at  Thu Sep 26 16:32:54 CEST 2019
